internal/container_linux_test.go

```diff
@@ -3,6 +3,8 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016 Datadog, Inc.
 
+//go:build test && linux
```

Comment:
The `test` build tag isn't defined, so these tests won't run. I think you just want the `linux` tag.

Reply:
Thanks. I just removed it !

I also asked a review to my team members. They should have the knowledge necessary. Normally I am just aligning on what is done in dogstatsd https://github.com/DataDog/datadog-go/pull/291